org.libvips.nip4.json

```diff
@@ -0,0 +1,343 @@
+{
+    "app-id" : "org.libvips.nip4",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nip4",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=host"
```

Comment:
I guess I forgot to ask but why do you need `host`. Previously this was home+media+srv but why?

Does this not work with filechooser portals?

Reply:
nip4 includes a macro language that lets you load files from any path, eg. `Image_file "/home/john/pics/x.jpg"`, meaning "Image constructor from a filename with path as argument". In turn this uses the libvips source system, which does not use portals.